src/test/resources/arquillian.xml

```diff
@@ -18,12 +19,33 @@
     <property name="attribute.pagenums" />
     <property name="attribute.toc" />
   </extension>
+
   <extension qualifier="asciidoctor-diagram">
     <property name="sourceDirectory">target/adoc-diagram-source</property>
     <property name="outputDirectory">target/adoc-diagram-target</property>
     <property name="backend">html</property>
     <property name="requires">asciidoctor-diagram</property>
   </extension>
+
+  <extension qualifier="asciidoctor-article">
```

Comment:
That is awesome, thanks for that, can you also update the readme maybe? Sounds like good samples no?